Source/Shared/BPUtils.m

```diff
@@ -121,6 +121,28 @@ + (NSError *)BPError:(const char *)function andLine:(int)line withFormat:(NSStri
                            userInfo:@{NSLocalizedDescriptionKey: msg}];
 }
 
++ (NSString *)findExecutablePath:(NSString *)execName {
+    NSString *argv0 = [[[NSProcessInfo processInfo] arguments] objectAtIndex:0];
+    NSString *execPath = [[argv0 stringByDeletingLastPathComponent] stringByAppendingPathComponent:execName];
+    if (![[NSFileManager defaultManager] isExecutableFileAtPath:execPath]) {
+        // Search the PATH for a bp executable
```

Comment:
Ugh, I'm not sure searching `PATH` is a good idea... we might find some other `bp` that is not even guaranteed to be from the same version of `bluepill`... I'm tempted to just take out this search and say that if we can't find it in the same path as `bluepill` then we can't find it.

For the `.dylib` we should also search `../lib/$exec`